packages/instant/src/containers/connected_account_payment_method.ts

```diff
@@ -56,7 +58,7 @@ const mergeProps = (
     ...ownProps,
     network: connectedState.network,
     account: connectedState.providerState.account,
-    walletName: connectedState.providerState.name,
+    walletDisplayName: connectedState.walletDisplayName || connectedState.providerState.name,
```

Comment:
I feel like this logic should be happening just once in the factory, but oh well :P